streams/src/test/java/org/apache/kafka/test/NoOpProcessorContext.java

```diff
@@ -29,18 +28,21 @@
 import org.apache.kafka.streams.processor.internals.AbstractProcessorContext;
 import org.apache.kafka.streams.processor.internals.MockStreamsMetrics;
 
+import java.time.Duration;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Properties;
 
 public class NoOpProcessorContext extends AbstractProcessorContext {
     public boolean initialized;
+    @SuppressWarnings("WeakerAccess")
```

Comment:
It looks like this field is never used outside of this class, which also implies that its value is never read. Should we just delete it?

Reply:
The `NoOpProcessorContext` is contained in `test` package and thus is built to be used in multiple tests. Thus it should be generic IMHO, even if we don't use all available "features". Thoughts?

Reply:
I just meant that the `forwardedValues` are being captured but never referenced. Therefore, we don't need to capture them. No need to maintain features in test support classes that no tests use.